connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaConfigBackingStoreTest.java

```diff
@@ -144,9 +146,21 @@
 
     private long logOffset = 0;
 
+    /**
+     * Using the invalid arguments to create a WorkerConfigTransformer is ok in testing since we override all public methods.
+     * TODO: implement the interface of WorkerConfigTransformer rather than doing the weird construction in testing
+     */
+    private final WorkerConfigTransformer transformer = new WorkerConfigTransformer(null, Collections.emptyMap()) {
+        @Override
+        public Map<String, String> transform(String connectorName, Map<String, String> configs) {
+            Assert.assertNotNull(configs);
+            return configs;
+        }
+    };
+
     @Before
     public void setUp() {
-        configStorage = PowerMock.createPartialMock(KafkaConfigBackingStore.class, new String[]{"createKafkaBasedLog"}, converter, DEFAULT_DISTRIBUTED_CONFIG, null);
+        configStorage = PowerMock.createPartialMock(KafkaConfigBackingStore.class, new String[]{"createKafkaBasedLog"}, converter, DEFAULT_DISTRIBUTED_CONFIG, transformer);
```

Comment:
Adding this transformer seems to break other tests in this file.  Perhaps leave it `null` for now.